understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
+          </figcaption>
+        </figure>
+
+        <p>Since a data table relies on two-dimensional layout for understanding, presenting the table in its own scrollable container would help ensure other content which does not meet an exception could reflow as the viewport adjusts.</p> 
+        
+        <figure>
+          <img src="img/table-exception.jpg" alt="a table within a containing element that provides bi-directional scrollbars to view the content of the table. The content above and below the table reflows to fit within the available width of the viewport.">
+          <figcaption>
+            <p>Tabular data has a Reflow exception, so a containing element to the table can provide bi-directional scrollbars, thus mitigating bi-directional scrollbars at the page level. On the left, a web page at default zoom shows a table of four columns represents the files a user has access to. On the right, the web page is zoomed in and the table is rendered within a containing element with is own bi-dimensional scrollbars.</p>
+          </figcaption>
+        </figure>
+
+        <p>There can be instances where bi-dimensional excepted content can extend beyond the visible viewport, and other non-excepted content exists, but does reflow within the visible viewport.  Unlike the prior table example with the paragraph not reflowing, such a situation does pass - but creates an unwanted user experience. For example, an unnecessary horizontal scrollbar could lead a user to believe there is content existing off-screen for them to scroll to. But if that scrollbar appears only because of one instance of excepted content, a user could expend effort looking for <em>other</em> content beyond that which reflows which doesn't exist.</p>
+
+        <figure>
+          <img src="img/unnecessary-horizontal-scrollbar.jpg" width="600" alt="BBC Earth video extends far beyond the horizontal width of the browser's viewport, resulting in a large horizontal scrollbar for the web page. The content of the web page otherwise fits into a 320px wide container, which is validated by use of a browser's developer tools which show the containing element has a width of 320px.">
+          <figcaption>
+            <p>A video player without a <code>max-width: 100%</code> CSS property causes it to remain full-sized at smaller (zoomed in) viewports. The rest of the page content reflows within 320 CSS pixel wide container. So, while there is a large horizontal scrollbar at the page level, it is due to the video player which has a reflow exception. Unlike the prior table example where the page was implemented in a way where paragraphs extended off-screen and required bi-directional scrolling to read, this scenario <em>passes</em> because all non-excepted content reflows. However, this does not the best user experience, and ideally the styling of the video player would update to mitigate the otherwise unnecessary horizontal scrollbar.</p>
```

Comment:
```suggestion
            <p>A video player without a <code>max-width: 100%</code> CSS property causes it to remain full-sized at smaller (zoomed in) viewports. The rest of the page content reflows within 320 CSS pixel wide container. So, while there is a large horizontal scrollbar at the page level, it is due to the video player which has a reflow exception. Unlike the prior table example where the page was implemented in a way where paragraphs extended off-screen and required bi-directional scrolling to read, this scenario <em>passes</em> because all non-excepted content reflows. However, this does not provide the best user experience, and ideally the styling of the video player would update to mitigate the otherwise unnecessary horizontal scrollbar.</p>
```